test/val/val_instructions_test.cpp

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) 2017 The Khronos Group Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Validation tests for illegal instructions
+
+#include <sstream>
+#include <string>
+#include <tuple>
+
+#include "gmock/gmock.h"
+#include "val_fixtures.h"
+
+using ::testing::HasSubstr;
+
+using ValidateIns = spvtest::ValidateBase<std::string>;
+
+namespace {
+
+TEST_F(ValidateIns, OpImageSparseSampleProjImplicitLod) {
+  char str[] = R"(
+             OpCapability Shader
+             OpCapability Linkage
```

Comment:
Changed to use OpEntryPoint and the image is left as an Input variable, but the other variables were changed to constants.